AudibleApi.Common/MetadataDtoV10.generated.cs

```diff
@@ -86,7 +89,6 @@ public partial class ContentReference
         public string Tempo { get; set; }
 
         [JsonProperty("version")]
-        [JsonConverter(typeof(ParseStringConverter))]
-        public long Version { get; set; }
```

Comment:
Have you checked to see if this new type compiles inside of Libation?

Reply:
It does. This value is audiobook file version and it's never referenced in Libation. The reason I'm changing it to string is because the api endpoint that uses it requires the value to be sent as a string. That suggests to me that it may not always parse to an integer.